provide rational for admin privileges (#32)
ni/install-systemlink-enterprise: getting-started/installation-guide.md

### 4.2 Install Cluster Prerequisites

> A cluster administrator with full access rights or a user with a cluster role that include privileges to create `CustomResourceDefinitions`, `ClusterRoles` and `ClusterRoleBindings` must use the systemlinkadmin Helm chart to install prerequisite resources globally on the cluster. This is due to the requirement for the [Strimzi Kafka Operator](https://strimzi.io/docs/operators/latest/deploying.html#deploying-cluster-operator-to-watch-whole-cluster-str) and [Jupyter Enterprise Gateway](https://jupyter-enterprise-gateway.readthedocs.io/en/v2.1.0/kernel-kubernetes.html#role-based-access-control-rbac) to create these objects.

Reviewer: ```suggestion
A cluster administrator with full access rights or a user with a cluster role that include privileges to create `CustomResourceDefinitions`, `ClusterRoles` and `ClusterRoleBindings` must use the systemlinkadmin Helm chart to install prerequisite resources globally on the cluster. The [Strimzi Kafka Operator](https://strimzi.io/docs/operators/latest/deploying.html#deploying-cluster-operator-to-watch-whole-cluster-str) and the [Jupyter Enterprise Gateway](https://jupyter-enterprise-gateway.readthedocs.io/en/v2.1.0/kernel-kubernetes.html#role-based-access-control-rbac) components create objects of these types during installation.
```

I'll defer to Melissa on voice, but I think this is cleaner?

Author: I think it't better. I'll commit the suggestion, but I would still like @MelissaHilliard to review.
